src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -133,23 +133,32 @@ trait MemberHandlers {
       else {
         // if this is a lazy val we avoid evaluating it here
         val resultString =
-          if (mods.isLazy) codegenln(false, "<lazy>")
-          else any2stringOf(path, maxStringElements)
+          if (mods.isLazy) """"""""
+          else """" = " + """ + any2stringOf(path, maxStringElements)
```

Comment:
I love the cartoon quality of the long eyebrow, but maybe these lines could be clearer? Anything that doesn't entail counting quotes.

Reply:
Oh, I can't tell by looking if it helps me understand what `lazy val i = 27` means.

Reply:
I've cleaned up the quotes, but right now the repl would print
```scala
scala> lazy val i = 27
lazy val i: Int

scala> 
```,

is `= <lazy>` noise or useful?

Reply:
`= ???`, perhaps

Reply:
what about
```
scala> lazy val x = 0
lazy val x: Int // unevaluated
```

Reply:
> `= ???`, perhaps

I'm concerned with triple question mark as that is a real value